x-pack/plugins/observability_solution/metrics_data_access/public/pages/link_to/use_asset_detail_redirect.test.tsx

```diff
@@ -57,13 +57,13 @@ describe('useNodeDetailsRedirect', () => {
     expect(getLinkProps).toHaveProperty('onClick');
   });
 
-  it('should return the LinkProperties for assetType host', () => {
-    const { result } = renderHook(() => useNodeDetailsRedirect(), { wrapper });
+  it('should return the LinkProperties for nodeType host', () => {
```

Comment:
Q: Why do we need to rename this from `assetType` to `nodeType`? I see we mostly use `asset` in this test/hook 🤔